src/main/java/com/google/codeu/data/Event.java

```diff
@@ -0,0 +1,116 @@
+package com.google.codeu.data;
```

Comment:
I will approve in advance, so that when you made the changes you would be able to merge. If you wanted me to review again let me know :)